### ISSUE TITLE
No way to pass `--experimental` to rattler-build

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/conda-smithy#2057]
I'm trying to [convert](https://github.com/conda-forge/zlib-feedstock/pull/83) zlib to rattler-build (tentatively towards emscripten support) broadly following [this](https://prefix.dev/blog/rattler_build_on_conda_forge). However, it turns out that some of the needed [functionality](https://prefix-dev.github.io/rattler-build/latest/multiple_output_cache/) is currently only available in rattler-build with an `--experimental` flag.

Smithy currently fails to rerender with the following
```
Error:   × Parsing: experimental only: `cache`.
```

AFAIU the main fix will need to happen inhttps://github.com/prefix-dev/rattler-build-conda-compat (which needs to define the API how to pass that on), and smithy will need to figure out how to allow that to be configured and pass it on.

Of course, one could say we don't want to support experimental features, but with stuff as fundamental as the global build cache, we don't have a choice, and I think it would be worthwhile to support as opt-in also going forward.

**Comment by bollwyvl** [comment]
Ran into this. How bad would a `conda_build_tool: rattler-build-experimental` be?

**Comment by wolfv** [comment]
yes, or help with the CEP :pray: we need to write and get it voted on.https://github.com/conda/ceps/pull/102
